build.sbt

```diff
@@ -1016,6 +1018,11 @@ lazy val `scio-smb`: Project = project
   .configs(
     IntegrationTest
   )
+  .dependsOn(
+    `scio-core`,
+    `scio-avro`,
+    `scio-test`
```

Comment:
Should it be this?
```suggestion
    `scio-test` % Test
```

Reply:
thanks! I forgot to clean up this and `scio-avro` after getting rid of the JobTest helper stuff.